beta/index.html

```diff
@@ -0,0 +1,627 @@
+            layout="nodisplay"
+            once=""
+        >
+        </amp-position-observer>
+        <p>Expressive Templates</p>
+        <ul>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/1.png"></amp-img></li>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/2.png"></amp-img></li>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/3.png"></amp-img></li>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/4.png"></amp-img></li>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/5.png"></amp-img></li>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/6.png"></amp-img></li>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/7.png"></amp-img></li>
+            <li><amp-img class="template" layout="responsive" width="542" height="840" src="./assets/templates/8.png"></amp-img></li>
+        </ul>
+    </section>
+
+     <section class="media-gallery">
+     </section>
+
+     <section class="faq">
+        <h2><a id="faq">FAQ</a></h2>
+        <dl>
+            <dt>Where can I learn more about Web Stories?</dt>
+            <dd><a href="https://amp.dev/about/stories/">Web Stories</a> are tappable, engaging visual stories brought to the web. They’re powered by AMP technology, so learn more about them on <a href="https://amp.dev/about/stories/">amp.dev</a>.</dd>
+        
+            <dt>How do I install the Stories for WordPress plugin?</dt>
+            <dd>
+                As soon as we’re graduating from beta, the plugin will be available on WordPress.org. While we’re in beta, the plugin has to be downloaded as zip. After that:
+
+                <ol>
+                    <li>Navigate to Plugins &gt; Add New.</li>
+                    <li>Click the Upload Plugin button at the top of the screen.</li>
+                    <li>Select the zip file from your local filesystem.</li>
+                    <li>Click the Install Now button.</li>
+                    <li>When installation is complete, you’ll see “Plugin installed successfully.” Click the Activate Plugin button at the bottom of the page.</li>
+                </ol>
+            </dd>
+        
+            <dt>I found a bug or missing feature! How do I report it?</dt>
+            <dd>Awesome! That’s exactly what the beta is for. Please submit feedback via <a href="https://docs.google.com/forms/d/e/1FAIpQLSfRK9gFWMavvGbEWPFUFz8PDgrDE6XdVuh3B9e1Fu549V8BxQ/viewform">this feedback form</a> or <a href="https://github.com/google/web-stories-wp/issues">file a bug or feature request on Github</a>. Your help is greatly appreciated.</dd>
```

Comment:
Would be nice to use `#3474E0` for the color of these links too.

Reply:
Good catch, done